modules/ROOT/pages/analytics-event-api.adoc

```diff
@@ -209,7 +235,12 @@ Your report can query data for one, many, or all of the available data fields. T
 |Violated Policy Name |The name of the policy violated by the API request (if any).
 |===
 
-If the Content-Length header is present, the Response Size is set to that value. If the Content-Length header is not present and the payload is a String, Analytics calculates the length of the String and reports that value. If the Content-Length header is not present and the payload is not a String, Analytics reports the response size as -1. For example, if the output returned is a DataWeave stream and the Content-Length header is not present, Analytics doesn't report a response size because the value is not a String. However if your application performs a String conversion, the response size is listed.
+In your query:
+
+* If the `Content-Length` header is present, the `Response Size` is set to that value. If the `Content-Length` header is not present and the payload is a String, Analytics calculates the length of the String and reports that value. 
+* If the `Content-Length` header is not present and the payload is not a String, Analytics reports the response size as -1. 
++
+For example, if the output returned is a DataWeave stream and the `Content-Length` header is not present, Analytics does not report a response size because the value is not a String. However if your application performs a String conversion, the response size is listed.
```

Comment:
```suggestion
For example, if the output returned is a DataWeave stream and the `Content-Length` header is not present, Analytics does not report a response size because the value is not a String. However, if your application performs a String conversion, the response size is shown.
```
I'm not sure that this example clarifies the three possible outputs.
Also, specify which "Analytics" you are referring to by using its complete name. There are several uses of Analytics in this text.

Reply:
Done! Thanks!